docs/ruleset_functions/is_fluid_loop_purchased_heating.md

```diff
@@ -0,0 +1,28 @@
+# is_fluid_loop_purchased_heating  
+
+**Description:** Returns TRUE if the fluid loop associated with the heating system associated with the HVAC system is attached to an external purchased heating loop. Returns FALSE if this is not the case.   
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the fluid loop associated with the heating system associated with the HVAC system is attached to an external purchased heating loop.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+
+**Returns:**  
+- **is_fluid_loop_purchased_heating**: Returns TRUE if the fluid loop associated with the heating system associated with the HVAC system is attached to an external purchased heating loop. Returns FALSE if this is not the case.   
+ 
+**Function Call:** None  
+
+## Logic:   
+- Check if RMR is not modeled with external fluid source: `if NOT RMR.ASHRAE229.external_fluid_source: is_fluid_loop_purchased_heating = FALSE`
+- Else, continue on: `Else:`  
+    - for each external fluid source in RMR: `for external_fluid_source.id in RMR.ASHRAE229.external_fluid_source:`
+        - Check if external fluid source is heating type: `if external_fluid_source.id.type == "HOT_WATER"` QUESTION: SHOULD IT IT BE OR STEAM?  
```

Comment:
good point. @mkarpman please confirm if it should be HW or steam.

Reply:
Yes, it should be either HW or steam. 
G3.1.1.1 Purchased Heat
For systems using purchased hot water or steam, the heating source shall be modeled as purchased hot water or steam in both the proposed design and baseline building design.

Reply:
Updated